src/disk_manager.rs

```diff
@@ -495,9 +522,7 @@ fn main() {
     }
     let log = Path::new(matches.value_of("log").unwrap());
     let backend = BackendType::from_str(matches.value_of("backend").unwrap()).unwrap();
```

Comment:
Unrelated to this change -- I am thinking we should write an improvement to get rid of these unwraps() in disk_manager

Reply:
Yeah we can fix those up.